packages/arb-token-bridge-ui/tests/e2e/specs/depositERC20.cy.ts

```diff
@@ -117,9 +202,82 @@ describe('Deposit ERC20 Token', () => {
                   symbol: 'WETH'
                 })}`
               ).should('be.visible')
+
+              // open the tx details popup
+              cy.findAllByLabelText('Transaction details button')
+                .first()
+                .click()
+                .then(() => {
+                  cy.findByText('Transaction details').should('be.visible')
+
+                  cy.findByText(/CUSTOM ADDRESS/i).should('be.visible')
+
+                  // custom destination label in pending tx history should be visible
+                  cy.findByLabelText(
+                    `Custom address: ${shortenAddress(
+                      Cypress.env('CUSTOM_DESTINATION_ADDRESS')
+                    )}`
+                  ).should('be.visible')
+                })
+
+              // close popup
+              cy.findByLabelText('Close transaction details popup').click()
             })
           })
       })
+
+      context('deposit should complete successfully', () => {
+        //wait for some time for tx to go through
+        cy.wait(60_000).then(() => {
```

Comment:
Could we instead open the settled transactions tab instantly and use `waitUntil` to wait for the specific ERC-20 amount row to show up?